libleptongui/include/gschem_options.h

```diff
@@ -132,24 +132,24 @@ GschemOptions*
 gschem_options_new ();
 
 void
-gschem_options_scale_snap_down (GschemOptions *options);
-
-void
-gschem_options_scale_snap_up (GschemOptions *options);
-
-void
-gschem_options_set_grid_mode (GschemOptions *options, GRID_MODE grid_mode);
-
+gschem_options_set_grid_mode (GschemOptions *options,
+                              SchematicGridMode grid_mode);
 void
 gschem_options_set_magnetic_net_mode (GschemOptions *options, gboolean enabled);
 
 void
 gschem_options_set_net_rubber_band_mode (GschemOptions *options, gboolean enabled);
 
 void
-gschem_options_set_snap_mode (GschemOptions *options, SNAP_STATE snap_mode);
+gschem_options_set_snap_mode (GschemOptions *options, SchematicSnapMode snap_mode);
 
 void
 gschem_options_set_snap_size (GschemOptions *options, int snap_size);
 
+SchematicSnapMode
+schematic_snap_mode_from_string (char *s);
+
+const char*
+schematic_snap_mode_to_string (SchematicSnapMode mode);
```

Comment:
`schematic_snap_mode_from_string()` and
`schematic_snap_mode_to_string()`
are already declared in `snap_mode.h`.

Reply:
I overlooked this.  Fixed, thanks!